tests/test_metrics.py

```diff
@@ -1,8 +1,9 @@
 import unittest
 import numpy as np
 import tensorflow as tf
-from training.metrics import count_flops,  count_trainable_params, InformationDensity, NetScore, count_flops_efficient
-
+from training.metrics import count_flops,  count_trainable_params, InformationDensity, NetScore, count_flops_efficient, get_type
+from upstride.type2.tf.keras import layers as type2_layers
+from upstride.type1.tf.keras import layers as type1_layers
```

Comment:
Upstride shouldn't be imported by default : user without upstride should be able to use this code

Reply:
Agreed. Removed them.